config/locales/en.yml

```diff
@@ -731,20 +731,6 @@ en:
       quantity: Quantity
 
   product_users:
-    index:
-      add_approved_user: Add Approved User
-      approval_explanation: |
-        This %{product_type} requires a user to be approved before they may
-        purchase the %{product_type}.
-      approved_users: Approved Users
-      confirm_removal: |
-        Are you sure you wish to remove this user from this %{product_type}?
-      no_users_approved: No users are currently approved.
-      th:
-        name: Name
-        user_name: User Name
-        email: Email
-      update: Update %{plural_label}
     new:
```

Comment:
I did not move this locale because I know it is overridden in UIC/DC (they don't use netids in their searches).